openedx/core/djangoapps/debug/views.py

```diff
@@ -40,10 +35,10 @@ def show_reference_template(request, template):
 
         # Add some messages to the course skeleton pages
         if u'course-skeleton.html' in request.path:
-            register_info_message(request, _('This is a test message'))
-            register_success_message(request, _('This is a success message'))
-            register_warning_message(request, _('This is a test warning'))
-            register_error_message(request, _('This is a test error'))
+            PageLevelMessages.register_info_message(request, _('This is a test message'))
```

Comment:
I really like this new class-based approach! It is simpler to import which makes it easier to work with, it is extensible, and the client code that registers the events is clearer to read too. Great stuff, @HarryRein and @robrap!